### PR TITLE
Enhancements/use profile image

### PR DESCRIPTION
**Pull request by JayyajGH**
The next round of using the profile image component

### CODE REVIEW

File: components/MyMessageReply.vue [hunk 1]
```diff
@@ -161,3 +134,22 @@ export default {
...
 }
 </script>
+
+<style scoped lang="scss">
+@import 'color-vars';
+
+.snippet {
+  border: 1px solid $color-gray--light;
+  border-radius: 10px;
+  padding-top: 2px;
```
Comment (aaronferris) [inline components/MyMessageReply.vue]: Could short hand this

File: components/ProfileImage.vue [hunk 2]
```diff
@@ -77,4 +77,14 @@ export default {
     height: 50px;
   }
 }
+
+.profile--xl {
+  width: 100px;
+  height: 100px;
+
+  @include media-breakpoint-up(md) {
```
Comment (aaronferris) [inline components/ProfileImage.vue]: Feels slightly odd to have a media query setting the same value as the default

File: pages/settings/index.vue [hunk 3]
```diff
@@ -824,3 +796,19 @@ export default {
   }
 }
 </script>
+
+<style scoped lang="scss">
+@import 'color-vars';
+
+.groupprofile {
+  height: 100px !important;
```
Comment (aaronferris) [inline pages/settings/index.vue]: Can we avoid the !important?

File: pages/settings/index.vue [hunk 4]
```diff
@@ -824,3 +796,19 @@ export default {
...
+}
+
+.nocardbot .card-body {
+  padding-bottom: 0px;
```
Comment (aaronferris) [inline pages/settings/index.vue]: Dont need to specify 'px' just '0' will do.

**Comment by JayyajGH** [comment]
@aaronferris - At the moment I've just replaced the profile image code with my new component.  I haven't refactored any of the old existing code in the components I've changed.  Although I agree with your comments I'm sort of loathe to make lots of random changes and increase the scope as part of this PR.  I don't even know if half of those old classes are still used.

I'll look at the media query one for the XL breakpoint as I think that is incorrect.

File: components/ProfileImage.vue (hunk 2 above)
Comment (JayyajGH) [reply components/ProfileImage.vue]: The image was 100px in all breakpoints before but I've now changed it to be 50px on smaller screens and 100px on larger screens.  This whole section needs some work though as there are existing layout issues that I haven't touched for now.

**Comment by edwh** [comment]
Lovely to see some crossreviewing happening, since you two are much 
better able to review each other's CSS than I am.

Agree not to fix too much creep stuff, but if appropriate please add to 
the shrapnel issue.

Where will the profile shrink in this fix?  There are some cases (like 
head of a chitchat thread) where it shouldn't be too small.


------ Original Message ------

**Comment by JayyajGH** [comment]
Yeah it's always good to have everything reviewed.  It makes you question your solution more thoroughly. 

The only size I changed from what it what originally is at the top of the settings page.  See what you think...

**Comment by edwh** [comment]
I'll have a look.  Ready to merge?

------ Original Message ------

**Comment by edwh** [comment]
I think it looks a bit small on mobile now, there.


------ Original Message ------
From: "Edward Hibbert" <edward@ehibbert.org.uk>
To: "Freegle/iznik-nuxt" 
<reply@reply.github.com>
Sent: 06/01/2020 11:40:05
Subject: Re[2]: [Freegle/iznik-nuxt] Enhancements/use profile image 
(#157)

**Comment by JayyajGH** [comment]
I'll bump it up to 75px then... And then it'll be ready to merge.

**Comment by JayyajGH** [comment]
@edwh - It's now slighter bigger on mobile

**Comment by JayyajGH** [comment]
I've added all of Aaron's comments to the shrapnel issue

**Pull request merged**